tests/python/python_structure_manager_test.py

```diff
@@ -134,7 +134,7 @@ def test_manager_iteration_2(self):
                 self.cutoff, **structure)
 
             for ii, center in enumerate(manager):
-                for jj, neigh in enumerate(center):
+                for jj, neigh in enumerate(center.pairs()):
```

Comment:
The `center.pairs` looks nice. But the enumerators are not used in some of the tests here. Can we get rid of them? I won't type then out all, but e.g. here:

```
            for ii, center in enumerate(manager):
                for jj, neigh in enumerate(center.pairs()):
                    dist = np.linalg.norm(neigh.position - center.position)
```